### DISCUSSION TITLE
Discord community call: 30 September 2023

### DISCUSSION THREAD

**Discussion by BenJamesBen** [freesewing/freesewing#5027] [Support] [answered]
You are all invited to our Discord hangout.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About

FreeSewing's Discord hangout is an informal hangout/chat/meeting that is open to everyone.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

### Details

Location: https://discord.freesewing.org/ (meeting room: 🔊voice chat)
Date: September 30th, 2023 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/sep-2-2023/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda

- TBD

**Comment by BenJamesBen** [comment]
I would like to add to the agenda:
- The removal of Unice from the FreeSewing repository. (Merged PR https://github.com/freesewing/freesewing/pull/5022)

**Comment by tangerineshark** [comment, marked as answer]
Not a ton of notes from this call, but a few general questions came up:

- If you post something on FS, who owns it, can make modifications, etc? For a lot of folks, this is the first / only open-source project they've worked on, adding to the lack of certainty on this question.
- What is the path for contributors to figure out how to contribute to FS? Is it just the Issues? And if something isn't flagged there, is the right solution to create an issue?
